_config.yml

```diff
@@ -18,9 +18,10 @@ navbar-links:
   Latest Roadmap: "https://www.bitcoincash.org/roadmap.html"
   Announcements: "announcements/"
   Download:
-    - Binaries: "https://download.bitcoinabc.org/"
+    - Binaries: "https://download.bitcoinabc.org/latest/"
```

Comment:
Personally, I'd change this label to "Latest Binaries", just for symmetry with "Archived Binaries". But not a show stopper.